python-libraries/data-platform-catalogue/data_platform_catalogue/client.py

```diff
@@ -79,13 +178,13 @@ class CatalogueClient:
     def __init__(
         self,
         jwt_token,
-        api_uri: str = "https://catalogue.apps-tools.development.data-platform.service.justice.gov.uk/api",
+        api_url: str = "https://catalogue.apps-tools.development.data-platform.service.justice.gov.uk/api",
```

Comment:
Can this be moved to config file

Reply:
Is this a config which sets env vars? Environment variables could make it easier to deploy a different `api_url` (etc) to different environments via GHA in future.

Reply:
This is just a default argument. The actual value used for client setup would be from usage of this catalogue package, and could be set by an env value.

Reply:
default string has been removed.